packages/blockprotocol/core.d.ts

```diff
@@ -25,6 +25,11 @@ export type BlockMetadata = {
   license?: string | null;
   name?: string | null;
   protocol?: string | null;
+  repository?: {
+    type: string;
+    url: string;
+    directory?: string;
+  };
```

Comment:
Can we also support the [shorthand syntax](https://docs.npmjs.com/cli/v7/configuring-npm/package-json#repository), a single string?